### PR TITLE
docs: update docsify documentation

### PR DESCRIPTION
**Pull request by Alex-Werner**
## Issue being fixed or feature implemented

As per our [CHANGELOG.md](https://github.com/dashevo/js-dpp/blob/2d0fba404dffc2b85ce553578fa3347742184984/CHANGELOG.md), we have some documentation to perform.

## What was done?
Updated docs : Buffer, Identifier, encodedProperties, .hash()...

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added or updated relevant unit/integration/functional/e2e tests
- [X] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -5,8 +5,10 @@
     - [dpp.identity](usage/identity.md)
     - [dpp.stateTransition](usage/stateTransition.md)
 - Primitives
+    - [AbstractStateTransition](primitives/AbstractStateTransition.md)
```
Comment (shumkov) [inline docs/_sidebar.md:8]: Should we list AbstractStateTransition here? It’s kinda abstract class. I guess we need to list just STs that inherited it?

Comment (Alex-Werner) [reply docs/_sidebar.md:8]: That's mostly for type explanation in parameters inputs. We refer to AbstractStateTransition. 
I recon we would want a way to be explicit in different part of them. But in this configuration, yeah we kinda need it.

**Review by shumkov — APPROVED** [review]
👍

**Comment by antouhou** [comment]
Looks great 👍

**Review by antouhou — APPROVED** [review]

**Pull request merged**